org.lflang/src/org/lflang/generator/LFGenerator.java

```diff
@@ -13,6 +13,7 @@
 import org.lflang.ErrorReporter;
 import org.lflang.FileConfig;
 import org.lflang.Target;
+import org.lflang.generator.c.CGenerator;
```

Comment:
Wait, why does LFGenerator now have a dependence on CGenerator?

Reply:
LFGenerator is responsible for instantiating the target generators. Thus it needs to import all generators. However, the imports don't appear here, because LFGenerator and most of the target generators are in the same package. But the CGenerator is now in the c subpackage and thus the import is needed.